packages/manager/core/api/src/client.ts

```diff
@@ -42,25 +43,37 @@ export const v6 = axios.create({
   ...defaultAxiosConfig,
   baseURL: '/engine/apiv6',
 });
-v6.interceptors.response.use(null, handleAuthenticationError);
 
 export const aapi = axios.create({
   ...defaultAxiosConfig,
   baseURL: '/engine/2api',
 });
-aapi.interceptors.response.use(null, handleAuthenticationError);
 
 export const ws = axios.create({
   ...defaultAxiosConfig,
   baseURL: '/engine/ws',
 });
-ws.interceptors.response.use(null, handleAuthenticationError);
 
 export const v2 = axios.create({
   ...defaultAxiosConfig,
   baseURL: '/engine/api/v2',
 });
-v2.interceptors.response.use(null, handleAuthenticationError);
 
-export const apiClient = { v6, aapi, ws, v2 };
+export const apiClient: any = { v6, aapi, ws, v2 };
```

Comment:
```suggestion
export const apiClient = { v6, aapi, ws, v2 };
```

Don't break the typing
Thank you